src/state-schemas/state-schema.js

```diff
@@ -111,6 +111,10 @@ export const stateProperties = {
     type: "boolean",
     default: false
   },
+  gettingNotebookHistory: {
```

Comment:
i see where this bit of state is set, but i'm not spotting where it is read/used?